code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -262,3 +262,17 @@
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/gun/energy,/obj/item/weapon/reagent_containers/spray/pepper,/obj/item/weapon/gun/projectile,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs)
 	armor = list(melee = 30, bullet = 15, laser = 30, energy = 10, bomb = 10, bio = 100, rad = 50)
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security
+
+/obj/item/clothing/head/helmet/space/hardsuit/security/hos
+	name = "head of security's hardsuit helmet"
+	desc = "a special bulky helmet designed for work in a hazardous, low pressure environment. Has an additional layer of armor."
+	icon_state = "hardsuit0-hos"
+	item_color = "hos"
+	armor = list(melee = 35, bullet = 20, laser = 30,energy = 10, bomb = 15, bio = 100, rad = 50)
+
+/obj/item/clothing/suit/space/hardsuit/security/hos
+	icon_state = "hardsuit-hos"
+	name = "head of security's hardsuit"
+	desc = "A special bulky suit that protects against hazardous, low pressure environments. Has an additional layer of armor."
+	armor = list(melee = 35, bullet = 15, laser = 30, energy = 10, bomb = 15, bio = 100, rad = 50)
```

Comment:
Up the bullet from 15 to 20 if you're going to do this; it doesn't make any sense for the bullet resist on the hardsuit to be less than the helmet; especially since most armors are consistent across pieces.